src/main/java/org/apache/maven/plugins/changes/announcement/AnnouncementMojo.java

```diff
@@ -665,6 +676,12 @@ protected List<Release> getJiraReleases() throws MojoExecutionException {
                         + " that matched the versionPrefix '" + versionPrefix + "'.");
             }
 
+            if (onlyCurrentVersion) {
+                String version = (versionPrefix == null ? "" : versionPrefix) + project.getVersion();
+                issueList = IssueUtils.getIssuesForVersion(issueList, version);
+                getLog().info("The JIRA Report will contain issues only for the current version.");
```

Comment:
probably should be debug level

Reply:
It is the same in jira report, by the way this should be also refactored .... 
`jiraDownloader` should does such work

Reply:
I'm not sure I follow. Is this message going directly into the report or is it on the console when the report generation runs?

Reply:
on console, changed to debug